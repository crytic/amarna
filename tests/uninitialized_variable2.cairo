func bar() -> (res):
    alloc_locals
    local x
    local xx
    %{
    ids.xx = 3
    %}
    return (res=x)
end