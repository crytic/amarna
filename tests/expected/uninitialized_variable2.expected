[dead-store] in uninitialized_variable2.cairo:4:11
[uninitialized-variable] in uninitialized_variable2.cairo:3:11
[unused-function] in uninitialized_variable2.cairo:1:1